### PR TITLE
fix: bell and minute rollover, add test case

### LINKED ISSUE #14: Time rollover is handled incorrectly
While trying to calculate some windows where the time, moon phase and weather all matched (for an S rank) I discovered that the windows returned were incomplete (when compared to other tools).

After further testing I found that the provided `weather_period` generator was returning 24, 16, and 8. When 24 is set as the bell in `EorzeaTime` it does not calculate the expected moon phase that would match the time and weather causing incorrect results. Now I am not an expert in time libraries (and do not wish to be) but 24 is an invalid hour (or bell in this case). 

The source is that the [`bell` setter allows 24](https://github.com/EltonChou/EorzeaEnv/blob/34e623e8a5d4bc188381db6a12f59608a0517ac9/EorzeaEnv/eorzea_time.py#L109) and the [`minute` setter allows 60](https://github.com/EltonChou/EorzeaEnv/blob/34e623e8a5d4bc188381db6a12f59608a0517ac9/EorzeaEnv/eorzea_time.py#L124), both should be `>=` instead of `>`. I would have submitted a PR but I was just going to make an issue to check if I was maybe missing something.

### PR DESCRIPTION
**Pull request by harbingerftw**
fixes #14 

I was unsure how much or little you wanted tested, does this work?

### CODE REVIEW

File: test/test_time.py [hunk 1]
```diff
@@ -60,6 +60,29 @@ def test_time_operator(self):
...
+        et.sun = 1
+        et.bell = 23
+        et.minute = 59
+
+        et.minute += 1
+
+        assert et.minute == 0
+        assert et.bell == 0
+        assert et.sun == 2
+        assert et.moon == 2
```
Comment (EltonChou) [inline test/test_time.py]: By default, EorzeaTime will use current timestamp to initialize.
`et.moon` will not be 2 in the future.

**Comment by EltonChou** [comment]
Recommend removing this part.
So you can check you tests' coverage and design your test cases.
https://github.com/EltonChou/EorzeaEnv/blob/34e623e8a5d4bc188381db6a12f59608a0517ac9/test/test_time.py#L75-L84

**Comment by harbingerftw** [comment]
I deleted the snippet you suggested, these tests cover all the branches now. does this look good?

File: test/test_time.py (hunk 1 above)
Comment (EltonChou) [reply test/test_time.py:74]: To ensure the result can be reproduced in future. Please specify timestamp to create EorzeaTime or assign a value to et.moon and et.year

File: test/test_time.py [hunk 2]
```diff
@@ -83,6 +83,37 @@ def test_rollover(self):
...
+        assert et.minute == 0
+        assert et.bell == 22
+        assert et.sun == 30
+        assert et.moon == 1
+
+        et.moon -= 11
+
+        assert et.moon == 2
+        assert et.year == 1069
+
```
Comment (EltonChou) [inline test/test_time.py]: To ensure the result can be reproduced in future. Please specify timestamp to create EorzeaTime or assign a value to `et.moon` and `et.year`

For example,
This test will be failed when `EorzeaTime(timestamp=datetime(2030, 1, 23).timestamp())`

**Review by EltonChou — COMMENTED** [review]
To ensure the result can be reproduced in future. Please specify timestamp to create EorzeaTime or assign a value to `et.moon` and `et.year`

For example,
Tests will be failed when `EorzeaTime(timestamp=datetime(2030, 1, 23).timestamp())`

**Comment by EltonChou** [comment]
LGTM
Thanks for you report and PR.

**Pull request merged**